nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/src/test/java/org/apache/nifi/web/security/jwt/key/service/StandardVerificationKeyServiceTest.java

```diff
@@ -38,13 +40,14 @@
 import java.util.Map;
 import java.util.UUID;
 
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-@RunWith(MockitoJUnitRunner.class)
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
```

Comment:
Is `LENIENT` necessary (Mockito default is `strict`)?

Reply:
I wouldn't consider it a blocker for a merge. AFAIK it doesn't actually create real problems aside from runtime bloat which isn't a critical issue for us. It's something that we could put in the backlog as a TODO for the whole code base as a cleanup activity.

Reply:
https://github.com/apache/nifi/pull/6813#discussion_r1059721468